### DISCUSSION TITLE
VRM - Daly BMS shows NaN:NaN ?

### DISCUSSION THREAD

**Discussion by centro-max** [Louisvdw/dbus-serialbattery#656] [Q&A]
The VRM Portal shows an additional Information "NaN:NaN" while charging the battery.

What does that mean? I think it is a Bug.

See the screenshot.

![Screenshot_2023-05-21-13-01-42-337_nl victronenergy~2](https://github.com/Louisvdw/dbus-serialbattery/assets/93882970/7e79f16e-7d37-406f-8706-51ed48432197)

Raspberry Pi 4b, Venus OS 3.00-42, 1.0.2...beta Serialbattery

**Comment by mr-manuel** [comment]
Could you try this version? https://github.com/mr-manuel/venus-os_dbus-serialbattery/releases/tag/v1.0.20230519beta

**Comment by centro-max** [comment]
Ok, installed v1.0.20230519beta, now everything is fine, but the error was previously only sporadic, not always. I couldn't see any regularity.

**Comment by centro-max** [comment]
No, the bug still exists.

![Fehler](https://github.com/Louisvdw/dbus-serialbattery/assets/93882970/374d97c1-a85e-4a2c-bae4-13f98259cb18)

**Reply by transistorgit** [reply]
This was seen also 
[1y ago](https://community.victronenergy.com/questions/103737/vrm-dashboard-secondary-battery-monitor-bmv-widget.html)

So probably not a daly driver problem. If you just want to get your display clean, you could disable "timetogo" in the config.ini of dbus-serialbattery.
